student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom:, name: 'Unknown', age: 0, parent_permission: true)
```

Comment:
- [ ] As I suggested above that age is not an optional parameter so if you change your Person class so it will effect its child classes too it would be nice if you update this constructor like below ⬇️

```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)
```